server/sonar-server/src/test/java/org/sonar/server/rule/ws/UpdateActionTest.java

```diff
@@ -310,6 +310,22 @@ public void fail_to_update_custom_when_description_is_empty() {
       .execute();
   }
 
+  @Test
+  public void throw_IllegalArgumentException_if_trying_to_update_builtin_rule_description() throws Exception {
+    logInAsQProfileAdministrator();
+    RuleDefinitionDto templateRule = db.rules().insert();
+
+    expectedException.expect(IllegalArgumentException.class);
+    expectedException.expectMessage("Not a custom rule");
+
+    ws.newRequest().setMethod("POST")
+      .setParam("key", templateRule.getKey().toString())
+      .setParam("name", "My custom rule")
```

Comment:
"My custom rule" is misleading as it suggest your updating a custom rule.
Please use rule.name(), it will be more easy to understand what the test is testing

Reply:
done